### PR TITLE
Added two new foods - Khinkali and Khachapuri

### PR DESCRIPTION
**Pull request by badbon**
I have added two new foods, Khinkali (Georgian dumpling) and (Adjaruli) Khachapuri (kind of a bread with cheese and egg), I believe there are lack of these kind of foods and it would be a nice and fun addition.

I have tested everything and it looks fine to me, however this is my first time working with DM and SS13 in general and first time making a PR on Github so sorry if I did something wrong - please correct me if anythings up.

Many thanks to OPDingo from coderbus for these great sprites (I think that was his name, I kind of forgot)

:cl:
rscadd: The chef can now make Khinkali and Khachapuri
/:cl:

### CODE REVIEW

**Comment by Man-Shroom-** [comment]
Unfortunately, /tg/ is in the middle of a FEATURE FREEZE, which means that only bugfix-type PR's will get merged. We exit the freeze on June 15th, so this will probably not get merged until then.

**Comment by NicholasM10** [comment]
Ah, alright. guess this'll be kept open until 15th?

I might add more commits then if that's okay.

**Comment by KorPhaeron** [comment]
I think we can probably let this slide since it isnt BALANCE and I dont want to crush a new coders spirit

**Comment by NicholasM10** [comment]
Aw, that'd be nice, I can even nerf these if needed, even though they are pretty low nutrition compared to similar foods, I think.

**Comment by Bawhoppen** [comment]
kor my spirit has been continously crused this freeze

**Comment by optimumtact** [comment]
@Bawhoppen in your case you deserve it :laughing:

File: code/modules/food&drinks/food/snacks_meat.dm [hunk 1]
```diff
@@ -124,6 +124,21 @@
...
 
+/obj/item/weapon/reagent_containers/food/snacks/rawkhinkali
+	name = "raw khinkali"
+	desc = "One hundred khinkalis? Do I look like a pig?"
+	icon_state = "khinkali"
+	list_reagents = list("nutriment" = 1, "vitamin" = 1)
+	cooked_type = /obj/item/weapon/reagent_containers/food/snacks/khinkali
+
+/obj/item/weapon/reagent_containers/food/snacks/khinkali
+    name = "khinkali"
```
Comment (Iamgoofball) [inline code/modules/food&drinks/food/snacks_meat.dm]: Please don't indent your lines using the spacebar, the TAB key is the codebase standard for indenting lines.

**Comment by Iamgoofball** [comment]
:+1: even had a changelog and picture

File: code/modules/food&drinks/recipes/tablecraft/recipes_pastry.dm [hunk 2]
```diff
@@ -170,6 +170,16 @@
 	result = /obj/item/weapon/reagent_containers/food/snacks/meatbun
 	category = CAT_FOOD
 
+/datum/crafting_recipe/food/khachapuri
+	name = "Khachapuri"
+	reqs = list(
+		/obj/item/weapon/reagent_containers/food/snacks/store/bread/plain = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/cheesewedge = 1,
+		/datum/reagent/consumable/eggyolk = 5
```
Comment (phil235) [inline code/modules/food&drinks/recipes/tablecraft/recipes_pastry.dm]: Put the reagents at the beginning of the list, otherwise bad things happen.

**Comment by optimumtact** [comment]
Fix the pointed out issues and this will be good to go :+1:

**Comment by NicholasM10** [comment]
@optimumtact 

Done that + tested it once again.

**Comment by optimumtact** [comment]
@NicholasM10 you missed @Iamgoofball's comment regarding spacing on the name variable

**Comment by NicholasM10** [comment]
@optimumtact 

Sorry I thought I was using tabs but I guess I accidently used space or something.

**Comment by optimumtact** [comment]
Congradulatiomiations :+1:

File: code/modules/food&drinks/recipes/tablecraft/recipes_meat.dm [hunk 3]
```diff
@@ -152,3 +152,11 @@
...
 	category = CAT_FOOD
 
+/datum/crafting_recipe/food/rawkhinkali
+        name = "Raw Khinkali"
+        reqs = list(
+                /obj/item/weapon/reagent_containers/food/snacks/doughslice = 1,
+                /obj/item/weapon/reagent_containers/food/snacks/faggot = 1
+        )
+        result =  /obj/item/weapon/reagent_containers/food/snacks/rawkhinkali
+        category = CAT_FOOD
```
Comment (Jordie0608) [inline code/modules/food&drinks/recipes/tablecraft/recipes_meat.dm]: This is all spaced not tabbed too

Comment (optimumtact) [inline code/modules/food&drinks/recipes/tablecraft/recipes_meat.dm]: damn it jordie

**Pull request merged**